apps/test/unit/templates/certificates/CertificateShareTest.js

```diff
@@ -31,28 +32,30 @@ describe('CertificateShare', () => {
   });
 
   it('renders announcement image url relative to pegasus', () => {
-    const wrapper = shallow(<CertificateShare {...defaultProps} />);
-
-    const printLink = wrapper.find('a');
-    expect(printLink.prop('href')).to.equal('/certificate-print');
-    const image = printLink.find('img');
-    expect(image.prop('src')).to.equal('/certificate-image');
-
-    const block = wrapper.find('TwoColumnActionBlock');
-    expect(block.prop('imageUrl')).to.equal('//code.org/announcement-image');
+    render(<CertificateShare {...defaultProps} />);
+
+    const printLink = screen.getByRole('link', {name: 'certificate alt text'});
+    expect(printLink.href).to.include('/certificate-print');
+    const image = screen.getByRole('img', {name: 'certificate alt text'});
+    expect(image.src).to.include('/certificate-image');
+
+    const twoColumnActionBlock = screen.getByTestId('two-column-action-block');
+    expect(twoColumnActionBlock).to.exist;
```

Comment:
Nit: I believe you don't need this line since `getBy` will throw an error and fail the test if `two-column-action-block` isn't found